config/supervisor/messenger-worker.ini

```diff
@@ -1,7 +1,7 @@
 [program:messenger-consume]
 command=php bin/console messenger:consume async --time-limit=3600
```

Comment:
Should I also add `--limit=10` (that means, the worker dies after having processed 10 messages from the queue.)